src/Symfony/Component/HttpKernel/KernelInterface.php

```diff
@@ -121,6 +121,8 @@ public function isDebug();
      * Gets the application root dir (path of the project's Kernel class).
      *
      * @return string The Kernel root dir
+     *
+     * @deprecated since Symfony 4.2, use getProjectDir() instead
```

Comment:
there's no `KernelInterface::getProjectDir` do we plan to add it? Perhaps update to `@see Kernel::getProjectDir()` otherwise.

Reply:
fixed